CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.11.0 (2023-05-12)
+
```

Comment:
we generally add this upgrade recommendation based on severity of the fixes, something like.

"This release contains bug fixes since the 2.10.3 release.
We recommend that you upgrade at the next available opportunity."

Reply:
as it also contains features; I've added:
```
This release contains new features and bug fixes since the 2.10.3 release.
We deem it moderate priority for upgrading.
```